ColabTool/Shared/PostFooter/index.jsx

```diff
@@ -57,9 +58,20 @@ const renderEdit = ({ onEditClick }) =>
   </span>;
 
 const renderApproval = ({
+  isPastDue,
   onApproveClick,
+  onEditClick,
   manager,
 }) => {
+  if (isPastDue) {
+    return (<span className={style['post-button-last']}>
+      <span className={style['vertical-line']} />
+      <Button onClick={onEditClick} noStyle>
```

Comment:
Wondering if this should be `onRescheduleClick`? The container that handles both `onEditClick` and `onRescheduleClick` could probably do the same thing. Seems like there could be a use case for having different events.